spec/rails_helper.rb

```diff
@@ -430,56 +449,37 @@ def synchronize(seconds = nil, errors: nil)
     # possible values: OFF, SEVERE, WARNING, INFO, DEBUG, ALL
     browser_log_level = ENV["SELENIUM_BROWSER_LOG_LEVEL"] || "WARNING"
 
-    chrome_browser_options =
-      Selenium::WebDriver::Chrome::Options
-        .new(logging_prefs: { "browser" => browser_log_level, "driver" => "ALL" })
-        .tap do |options|
-          apply_base_chrome_options(options)
-          options.add_argument("--disable-search-engine-choice-screen")
-          options.add_argument("--window-size=1400,1400")
-          options.add_preference("download.default_directory", Downloads::FOLDER)
-        end
-
-    driver_options = { browser: :chrome, timeout: BROWSER_READ_TIMEOUT }
-
-    if ENV["CAPYBARA_REMOTE_DRIVER_URL"].present?
-      driver_options[:browser] = :remote
-      driver_options[:url] = ENV["CAPYBARA_REMOTE_DRIVER_URL"]
-    end
-
-    desktop_driver_options = driver_options.merge(options: chrome_browser_options)
-
-    Capybara.register_driver :selenium_chrome do |app|
-      Capybara::Selenium::Driver.new(app, **desktop_driver_options)
-    end
-
-    Capybara.register_driver :selenium_chrome_headless do |app|
-      chrome_browser_options.add_argument("--headless=new")
-      Capybara::Selenium::Driver.new(app, **desktop_driver_options)
+    driver_options = {
+      browser_type: :chromium,
+      channel: :chrome,
+      headless: ENV["SELENIUM_HEADLESS"] != "0",
+      args: apply_base_chrome_args,
+      env: [{ name: browser_log_level, value: "pw:browser" }],
+      acceptDownloads: true,
+      downloads_path: Downloads::FOLDER,
+      devtools: ENV["CHROME_DEV_TOOLS"].present?, # playwright recomments https://playwright.dev/docs/debug instead
```

Comment:
we can't define the position as we used to do... but I think that's not worth spending time to dig this